### PR TITLE
Fix fixtures

### PR DESCRIPTION
**Pull request by longfin**

### CODE REVIEW

File: .Lib9c.Tests/xunit.runner.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+        "$schema": "https://xunit.net/schema/current/xunit.runner.schema.json",
+        "maxParallelThreads": 32
+}
```
Comment (greymistcube) [inline .Lib9c.Tests/xunit.runner.json]: Is this change intentional?

File: .Lib9c.Tests/Lib9c.Tests.csproj [hunk 2]
```diff
@@ -57,5 +57,7 @@
                when the configuration is 'DevEx'.-->
     <ProjectReference Include="..\Lib9c.DevExtensions\Lib9c.DevExtensions.csproj" />
   </ItemGroup>
-
+  <ItemGroup>
+    <Content Include="xunit.runner.json" CopyToOutputDirectory="PreserveNewest" />
+  </ItemGroup>
```
Comment (greymistcube) [inline .Lib9c.Tests/Lib9c.Tests.csproj]: Is this change intentional?

**Comment by greymistcube** [comment]
As the intention of introducing `MockState` was to have clean `IAccountStateDelta` before execution, I'll add few changes before merging this. 😗

This should make things significantly easier to fix failing tests (at least for using a copy of `AccountStateDelta` from libplanet). 😀

File: .Lib9c.Tests/xunit.runner.json (hunk 1 above)
Comment (longfin) [reply .Lib9c.Tests/xunit.runner.json]: Nope, it seems a mistake.

Comment (longfin) [reply .Lib9c.Tests/xunit.runner.json]: Amended. https://github.com/greymistcube/lib9c/pull/1/commits/b86894bb798dc07a9668d5e04a5d26b7ec288576

File: .Lib9c.Tests/Lib9c.Tests.csproj (hunk 2 above)
Comment (longfin) [reply .Lib9c.Tests/Lib9c.Tests.csproj]: Amended. https://github.com/greymistcube/lib9c/pull/1/commits/b86894bb798dc07a9668d5e04a5d26b7ec288576
